app/controllers/application_controller.rb

```diff
@@ -1,20 +1,31 @@
 class ApplicationController < ActionController::Base
-  before_action :debug_request
+  before_action :set_tenant, :debug_request
   set_current_tenant_through_filter
-  before_action :set_tenant
-
-  def debug_request
-    logger.debug("SUBDOMAIN: #{request.subdomain}")
-    logger.debug("TENANT: #{ActsAsTenant.current_tenant}")
-  end
 
   def set_tenant
     org = Organization.find_by(subdomain: request.subdomain)
     set_current_tenant(org)
+    @organization_settings = get_organization_settings[request.subdomain]
+  end
+
+  def debug_request
+    logger.debug("SUBDOMAIN: #{request.subdomain}")
+    logger.debug("TENANT: #{ActsAsTenant.current_tenant}")
   end
 
   private
 
+  def get_organization_settings
+    {
+      "alta" => {
+        logo: "logo.png"
+      },
+      "rubyforgood" => {
+        logo: "rubyforgood-logo.png"
+      }
+    }
+  end
```

Comment:
Hmm maybe we could instead use a attachment on Organization with ActiveStorage. So like `organization.logo`

Reply:
Technically I think we could use rich text columns for the various pages on the Organization to get that wysiwyg experience. It won't be super super customizable but enough to maybe be different enough?